analysis/webservice/algorithms/doms/BaseDomsHandler.py

```diff
@@ -293,254 +313,68 @@ def create(executionId, results, params, details):
         dataset.geospatial_lon_resolution = "point"
         dataset.geospatial_lat_units = "degrees_north"
         dataset.geospatial_lon_units = "degrees_east"
-        dataset.geospatial_vertical_min = 0.0
-        dataset.geospatial_vertical_max = params["radiusTolerance"]
+        dataset.geospatial_vertical_min = float(params["depthMin"])
+        dataset.geospatial_vertical_max = float(params["depthMax"])
         dataset.geospatial_vertical_units = "m"
         dataset.geospatial_vertical_resolution = "point"
```

Comment:
Come to think of it `dataset.geospatial_vertical_resolution = "point"` doesn't make sense either.